### ISSUE TITLE
@wordpress/edit-site: Consider exporting Global Styles preview cards.

### ISSUE THREAD

**Issue by StevenDufresne** [WordPress/gutenberg#44886]
## What problem does this address?
The Global Styles [preview card](https://github.com/WordPress/gutenberg/blob/trunk/packages/edit-site/src/components/global-styles/preview.js#L105) is useful for promoting style variations. In order to preview theme style variations on wordpress.org we had to recreate the component. This is a short-term solution and is bound to fall out of sync. It would be ideal if this component were exportable from the `@wordpress/edit-site`.

You can read more about the context here:
https://meta.trac.wordpress.org/ticket/6059

## What is your proposed solution?
I recognize that the card is not currently exportable because it relies on local contexts and hooks, but we could probably create another component that receives the global styles and export that from the package, starting at the [iframe](https://github.com/WordPress/gutenberg/blob/trunk/packages/edit-site/src/components/global-styles/preview.js#L105) tag.

I would like the ability to export it as a component:
```
import { StylePreview } from '@wordpress/edit-site' 

...

<StylePreview {...TBD props}  />
```
Ideally, it does not require more `@wordpress/{package_name}` packages and is very lightweight.

## Screenshot:
<img width="125" alt="Screen Shot 2022-10-12 at 10 03 32 AM" src="https://user-images.githubusercontent.com/1657336/195226328-b7d3c9ab-219e-4453-aacb-ce99e607a773.png">

**Comment by talldan** [comment]
I see there are some components exported from `@wordpress/edit-site`, but I don't think it's a good pattern to follow. It would be better to look at making this a generic component that resides in the `@wordpress/block-editor` or `@wordpress/components`.

`@wordpress/edit-site` is more of an application entry point, so the problem with using it as a dependency is that it naturally has lots of dependencies itself, and so is quite a lot to install and bundle (if tree shaking doesn't work).

IIRC there was also a plan to revise the design of this button, but I can't find an issue for it.

**Comment by StevenDufresne** [comment]
Yep, that works as well. The button recently went through updates so it's probably closed.

**Comment by jasmussen** [comment]
> IIRC there was also a plan to revise the design of this button, but I can't find an issue for it.

If it's the issue I'm thinking of (can't find it either, but it was an effort to show more variation in the card), it has been solved in recent iterations, and I would expect the current design to be pretty stable for the foreseeable future.

**Comment by youknowriad** [comment]
Closing for low demand. We can resurface if needed at some point.
